dataprofiler/tests/data_readers/test_data.py

```diff
@@ -14,6 +16,60 @@ def test_read_data(self):
         for data_type, none_type in zip(data_types, none_types):
             Data(data=none_type, data_type=data_type)
 
+class TestDataReadFromURL(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        test_dir = os.path.join(test_root_path, 'data')
+        cls.input_file_names = [
+            dict(path=os.path.join(test_dir, 'csv/diamonds.csv'),
+                 count=53940, delimiter=',', has_header=[0],
+                 num_columns=10, encoding='utf-8', data_type='csv'),
+            dict(path=os.path.join(test_dir, 'avro/users.avro'), count=4,  data_type='avro'),
+            dict(path=os.path.join(test_dir, 'json/iris-utf-16.json'), encoding='utf-16', count=150, data_type='json'),
+            dict(path=os.path.join(test_dir, 'parquet/iris.parq'), count=150, data_type='parquet'),
+            dict(path=os.path.join(test_dir, 'txt/code.txt'), count=150, data_type='text'),
+        ]
+
+    @mock.patch('requests.get')
+    def test_read_from_url(self, mock_request_get):
+        """Ensure error logs for trying to save empty data."""
+        def chunk_file(filepath, c_size):
+            with open(filepath, 'rb') as fp:
+                bytes = fp.read(c_size)
+                while bytes != b"":
+                    yield bytes
+                    bytes = fp.read(c_size)
+
+        for input_file in self.input_file_names:
+            mock_request_get.return_value.__enter__.return_value.iter_content.side_effect = \
+                lambda chunk_size: chunk_file(input_file['path'], chunk_size)
+            data_obj = Data('https://test.com')
+            self.assertEqual(data_obj.data_type, input_file['data_type'])
```

Comment:
What exactly is this testing? Because this url isn't real so why would we expect the data_types to match all of the files in `self.input_file_names`? Some comments here would be really helpful.